### ISSUE TITLE
support passing entire definition to component

### ISSUE THREAD

**Issue by tomwayson** [Esri/ember-cli-cedar#55]
`{{cedar-chart definition=model}}`

**Comment by tomwayson** [comment]
probably best way to deal w/ this is to go back to `new Cedar()` taking `options` instead of a `definition`.

So it would look like:
```js
if (options.definition) {
  this.definition(options.definition)
}
if (options.datasets) {
  this.datasets(datasets)
}
if (options.series) {
  this.series(series)
}
...
```

**Comment by tomwayson** [comment]
@benstoltz  and I agreed that we're not going to get back into that ^^^ confusing business where the chart constructor `options` is pretty much a `definition` (has `datasets`, `series`, etc) but it [can optionally have a `definition` property](https://github.com/Esri/cedar/blob/09c45e48a039098c65b17ed427e1d735fbf00fb3/src/cedar.js#L139)

Instead the component here will handle the logic of creating a definition from it's properties, using  `Object.assign` if a `definition` and other props were passed.

**Comment by tomwayson** [comment]
resolved in #56
